src/main/java/net/neoforged/neoforge/server/command/generation/GenerationTask.java

```diff
@@ -209,7 +209,7 @@ private void acquireChunk(long chunk) {
 
     private void releaseChunk(long chunk) {
         ChunkPos pos = new ChunkPos(chunk);
-        this.chunkSource.addRegionTicket(NEOFORGE_GENERATE_FORCED, pos, 0, pos);
+        this.chunkSource.removeRegionTicket(NEOFORGE_GENERATE_FORCED, pos, 0, pos);
```

Comment:
Yep, the key for the ticket is the ChunkPos object. Which I was remaking which means the key doesn't match the same key used for adding... Will adjust code to pass same exact chunkPos object to removal

Reply:
or maybe not. Should be comparing internally on longs anyway. Still getting leak even with same chunkPos object